pkgs/applications/misc/phoc/default.nix

```diff
@@ -15,36 +15,53 @@
 , libdrm
 , libxkbcommon
 , wlroots
+, runCommand
+, patchutils
+, fetchurl
 }:
 
 let
-  phocWlroots = wlroots.overrideAttrs (old: {
-    patches = (old.patches or []) ++ [
-      # Temporary fix. Upstream report: https://source.puri.sm/Librem5/phosh/-/issues/422
-      (fetchpatch {
-        name = "0001-Revert-layer-shell-error-on-0-dimension-without-anch.patch";
-        url = "https://gitlab.alpinelinux.org/alpine/aports/-/raw/78fde4aaf1a74eb13a3f083cb6dfb29f578c3265/community/wlroots/0001-Revert-layer-shell-error-on-0-dimension-without-anch.patch";
-        sha256 = "1zjn7mwdj21z0jsc2mz90cnrzk97yqkiq58qqgpjav4h4dgpfb38";
-      })
```

Comment:
This patch is still needed, along with another revert (haven't tested). See [the commit history](https://source.puri.sm/Librem5/wlroots/-/commits/2fce64d30d378d7009a5770b2472231a0e535ada/) of the forked version they are pinning onto.